docs-new/docs/graphql/core/api-reference/syntax-defs.mdx

```diff
@@ -241,229 +213,204 @@ String
 | delete           | false    | `String` | Customise the `delete_<table-name>` root field       |
 | delete_by_pk     | false    | `String` | Customise the `delete_<table-name>_by_pk` root field |
 
-## Custom Function Root Fields
+## Custom Function Root Fields {#customfunctionrootfields}
 
 | Key                | Required | Schema   | Description                                          |
 |--------------------|----------|----------|------------------------------------------------------|
 | function           | false    | `String` | Customise the `<function-name>` root field           |
 | function_aggregate | false    | `String` | Customise the `<function-name>_aggregete` root field |
 
-## InsertPermission
+## InsertPermission {#insertpermission}
 
 | Key          | Required | Schema                      | Description                                                                                                                                                                                                               |
 |--------------|----------|-----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| check        | true     | `BoolExp`                   | This expression has to hold true for every new row that is inserted                                                                                                                                                       |
-| set          | false    | `ColumnPresetExp`           | Preset values for columns that can be sourced from session variables or static values                                                                                                                                     |
-| columns      | false    | `PGColumn` array (or) `'*'` | Can insert into only these columns (or all when `'*'` is specified)                                                                                                                                                       |
+| check        | true     | [BoolExp](#boolexp)                   | This expression has to hold true for every new row that is inserted                                                                                                                                                       |
+| set          | false    | [ColumnPresetExp](#columnpresetexp)           | Preset values for columns that can be sourced from session variables or static values                                                                                                                                     |
```

Comment:
both id and label mismatch, missed `s`, ColumnPreset`s`

`[ColumnPresetsExp](#columnpresetsexp)`

Reply:
Label is ColumnPresetsExp and id is #columnpresetexp in rst file :(

Reply:
oh my bad, then lets change label only